WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -383,24 +383,14 @@ def getJobAttributes(self, jobID, attrList=None):
       return ret
     jobID = ret['Value']
 
-    if attrList:
```

Comment:
Why is this whole escaping not necessary anymore ?

Reply:
The code was duplicated, first with `attrList`, then for `self.jobAttributesName`.
I replaced it ni line 387 with a selection between the two depending on whether `attrList` is set or no